bulk_site_creator/utils.py

```diff
@@ -38,11 +38,11 @@ def generate_task_objects(course_instances: list[dict], job: JobRecord):
                 course_code = ci.course.registrar_code
 
             if ci.course.course_group:
-                sis_account_id = f"sis_account_id:coursegroup:{ci.course.course_group_id}"
+                sis_account_id = ci.course.course_group_id
             elif ci.course.department:
-                sis_account_id = f"sis_account_id:department:{ci.course.department_id}"
+                sis_account_id = ci.course.department_id
             else:
-                sis_account_id = f"sis_account_id:school:{ci.course.school_id}"
+                sis_account_id = ci.course.school_id
```

Comment:
You will still want to keep the :department, :coursegroup and :school prefixes however